SPECS/ca-certificates/ca-certificates.spec

```diff
@@ -44,7 +44,7 @@ Name:           ca-certificates
 
 # When updating, "Epoch, "Version", AND "Release" tags must be updated in the "prebuilt-ca-certificates*" packages as well.
 Epoch:          1
-Version:        3.0.0
+Version:        %{azl}.0.0
```

Comment:
Why do we have this set to 3.0.0 instead of 3.0?

Reply:
We wanted to use semantic versioning. I'm actually curious why some of our packages have only two version numbers.